README.md

```diff
@@ -2,10 +2,25 @@
 
 A Bevy CLI tool.
```

Comment:
This needs more expansion to describe the current and planned features. This is the first thing people will read!

Reply:
I wasn't sure about what other planned features we have, so I just rewrote the opening description and added a list of current features. Tell me if there's anything I missed in fc10f2f4888f291363e7bfe5f18a9d9add942322!